src/frontend/komponenter/Fagsak/Høyremeny/Totrinnskontroll/Totrinnskontroll.tsx

```diff
@@ -74,47 +74,54 @@ const Totrinnskontroll: React.FC = () => {
-                                                    e.preventDefault()
-                                                }
-                                                onClick={() => navigerTilSide(side as ISide)}
-                                            >
-                                                {behandlingssteg[totrinnSteg.behandlingssteg]}
-                                            </Link>
-                                        }
-                                        value={
-                                            !erLesevisning
-                                                ? totrinnSteg.godkjent
-                                                : totrinnSteg.godkjent === OptionGodkjent
-                                                  ? 'Godkjent'
-                                                  : totrinnSteg.godkjent === OptionIkkeGodkjent
-                                                    ? 'Vurder på nytt'
-                                                    : 'Ikke vurdert'
-                                        }
-                                        onChange={(val: TotrinnGodkjenningOption) =>
-                                            oppdaterGodkjenning(totrinnSteg.index, val)
-                                        }
-                                        error={
-                                            totrinnSteg.feilmelding ? totrinnSteg.feilmelding : null
-                                        }
-                                    >
-                                        {totrinnGodkjenningOptions.map(opt => (
-                                            <Radio
-                                                key={opt.label}
-                                                name={`stegetGodkjent_${totrinnSteg.index}`}
-                                                data-testid={`stegetGodkjent_${totrinnSteg.index}-${opt.verdi}`}
-                                                value={opt}
-                                            >
-                                                {opt.label}
-                                            </Radio>
-                                        ))}
-                                    </HorisontalFamilieRadioGruppe>
+                                    <Label>
+                                        <Link
+                                            href="#"
+                                            onMouseDown={(e: React.MouseEvent) =>
+                                                e.preventDefault()
+                                            }
+                                            onClick={() => navigerTilSide(side as ISide)}
+                                        >
+                                            {behandlingssteg[totrinnSteg.behandlingssteg]}
+                                        </Link>
+                                    </Label>
+                                    {erLesevisning ? (
```

Comment:
Kan vi bruke lesevisningen til readoknappene? 
https://aksel.nav.no/komponenter/core/radio#radiodemo-readonly

Reply:
Radioknappene kan kun velge mellom "godkjent" og "vurder på nytt", men vi trenger "ikke vurdert" som et gyldig visningsalternativ for lesevisning